web/app/themes/ppj/page-footer.php

```diff
@@ -34,5 +34,7 @@
             <?php endif; ?>
         </div>
     </div> <?php // page-container ?>
+    <?php include 'partials/mobileNavigation.php'; ?>
```

Comment:
Can you use the `partial()` function here instead of a straight include? To keep it consistent with how partials are included elsewhere.

I'm thinking something like:

```php
<?php echo ppj\partial('mobileNavigation'); ?>
```

Reply:
Done.